lib/portlayer/exec/container.go

```diff
@@ -133,28 +166,6 @@ func GetContainer(ctx context.Context, id uid.UID) *Handle {
 	return nil
 }
 
-func (s State) String() string {
-	switch s {
-	case StateCreated:
-		return "Created"
-	case StateStarting:
-		return "Starting"
-	case StateRunning:
-		return "Running"
-	case StateRemoving:
-		return "Removing"
-	case StateRemoved:
-		return "Removed"
-	case StateStopping:
-		return "Stopping"
-	case StateStopped:
-		return "Stopped"
-	case StateUnknown:
-		return "Unknown"
-	}
-	return ""
-}
-
 // CurrentState returns current state.
 func (c *Container) CurrentState() State {
 	c.m.Lock()
```

Comment:
in my related PR I switched over to use atomics for the state. It will not block when there is the only interest to get a current state for info purposes.